### ISSUE TITLE
Sync Issue

### ISSUE THREAD

**Issue by morquis** [MichaBrugger/booksidian_plugin#12]
When I add a book to an additional shelve on Goodreads, the Obsidian file is not updated, even as the RSS feed shows the additional shelve.

**Comment by MichaBrugger** [comment]
That's true, currently I do not update any files (I just check that they exist). The reason for that is that it gets very complex very quickly when I start updating existing files. Let's say you have written a summary and some notes into the file or you've have updated the frontmatter in some sort of way, I then need to distinguish between content that was added by you on purpose and content that should be updated because it's outdated.

I agree that this would be a great feature and I understand your request, but given my current schedule I probably won't be able to implement such a logic anytime soon. I hope you can understand. That being said, if anyone would like to build this, my projects are always open for anyone wanting to contribute :)

**Comment by mjh8431** [comment]
morquis - are you wanting to update existing files or simply to add the new file to you folder? Also, have you added the additional shelf to the Shelves part of the Setup?

**Comment by morquis** [comment]
I want to update the existing file.
Also I have added the shelf and I'm aware of the 100 per shelf restriction.
When I add a new book to Goodreads "reading" shelf, it gets switched to "read" and removed from "reading", when I finish it.
I would like to have this change reflected in the "synced" files.
The books are fetched not via read, but from other shelves, where I can satisfy the <100 rule.

**Comment by zacharyw** [comment]
@morquis I believe your issue should be handle-able in the new 0.3.0 version of the plugin. There's a new plugin setting called "overwrite" (off by default). If you toggle that on, incoming data with overwrite previous notes. So if you move something to a new shelf, it will replace the previous note with the new one.

Just be careful it does completely replace the note - if you were putting your own notes in those files they will be overwritten.
